src/main/java/org/apache/mesos/hdfs/config/HdfsConfig.java

```diff
@@ -1,22 +1,58 @@
 package org.apache.mesos.hdfs.config;
 
 import com.google.inject.Singleton;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.conf.Configured;
 import org.apache.hadoop.fs.Path;
+import org.apache.mesos.hdfs.util.HDFSConstants;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.util.Properties;
 
+/**
+ * Provides executor configurations for launching processes at the slave leveraging hadoop
+ * configurations
+ */
 @Singleton
-public class SchedulerConf extends Configured {
+public class HdfsConfig {
 
-  public SchedulerConf(Configuration conf) {
+  private Configuration hadoopConfig;
```

Comment:
Can we name this Configuration variable hdfsConfig instead of hadoopConfig? Then maybe the class could be named HdfsConfigDefinition or HDFSSchedulerConfig